src/main/Validator.ts

```diff
@@ -102,6 +102,7 @@ const configDataSchemaV3 = Joi.object<ConfigV3>({
         tabs: Joi.array().items(Joi.object({
             name: Joi.string().required(),
             order: Joi.number().integer().min(0),
+            isClosed: Joi.boolean().default(false),
```

Comment:
shouldn't this depend on whether it is focalboard or not?

Reply:
I don't think that matters, we want to be able to close any tab theoretically, but we're currently just disallowing the closing of Messages. We want each tab type to be written such that it controls whether it's closable or not, but I don't think we need to limit that here, it should just be ignored if you tried to close a tab that isn't closable.

Reply:
True, I don't think it does any harm.